hopp/simulation/technologies/pv/pv_plant.py

```diff
@@ -133,4 +147,60 @@ def dc_degradation(self) -> float:
     @dc_degradation.setter
     def dc_degradation(self, dc_deg_per_year: Sequence):
         """Sets annual DC degradation for lifetime simulations [%/year]."""
-        self._system_model.Lifetime.dc_degradation = dc_deg_per_year
\ No newline at end of file
+        self._system_model.Lifetime.dc_degradation = dc_deg_per_year
+
+    @property
+    def dc_ac_ratio(self) -> float:
+        """DC to AC inverter loading ratio [ratio]."""
+        return self._system_model.SystemDesign.dc_ac_ratio
+
+    @dc_ac_ratio.setter
+    def dc_ac_ratio(self, inverter_loading_ratio: float):
+        """Sets DC to AC inverter loading ratio [ratio]."""
+        self._system_model.SystemDesign.dc_ac_ratio = inverter_loading_ratio
+    
+    @property
+    def module_type(self) -> int:
+        """ Module type: standard, premium, thin film [0/1/2]"""
+        return self._system_model.value("module_type")
+
+    @module_type.setter
+    def module_type(self, solar_module_type: int):
+        """ Sets module type: standard, premium, thin film [0/1/2]"""
+        if 0 <= solar_module_type <= 2:
+            self._system_model.value("module_type", solar_module_type)
+        else:
+            raise ValueError("Invalid module type")
+        if solar_module_type is not None:
+            efficiency_mapping = {0: 0.19, 1: 0.21, 2: 0.18}
+            if solar_module_type in efficiency_mapping:
+                self.approx_nominal_efficiency = efficiency_mapping[solar_module_type]
+            else:
+                raise ValueError("Module type not recognized")
+        else:
+            raise NotImplementedError("Module type not set")
+
+    @property
+    def plant_area(self):
+        """Estimated Total Module Area [m2]"""
+        if self.approx_nominal_efficiency == 0:
+            raise ValueError("approx_nominal_efficiency cannot be zero.")
+        return self._system_model.SystemDesign.system_capacity / self.approx_nominal_efficiency
```

Comment:
I don't understand why the area is the system capacity divided by an efficiency? Shouldn't it be a kw per m2 value?

Reply:
I agree the units don't make sense, which is what I had messaged you about via Teams. This is based on an estimate in the SAM GUI for pvwatts8, we talked about getting a more specific area calculation for each module type in the future but thought this would be alright in the mean time.

Reply:
Could you use 11.092 kg/m2 as the default value, taken from the SAM Transient Thermal Model? There's even a reference if you need to explain it: Prilliman, M.; Stein, J. S.; Riley, D.;  Tamizhmani, G. (2020). [Transient Weighted Moving-Average Model of Photovoltaic Module Back-Surface Temperature](https://ieeexplore.ieee.org/document/9095219). IEEE Journal of Photovoltaics. 8 pp. Volume 10 No. 4.

Reply:
I updated the mass calculation in https://github.com/NREL/HOPP/pull/246/commits/66d87e6f01722ac954a62d8d397d912f12532bea to use 11.092kg/m2.

Are you interested in me updating the area calculation to a different method?

Reply:
Oh yes, area! There's a file `pv_module.py` in HOPP that contains the PVWatts default module that has `module_width` and `module_height` and `module_power` so you could use that to set up the multiplier

Reply:
Updated to use `get_module_attribs` to get PVWatts `module_area` and `module_power` in https://github.com/NREL/HOPP/pull/246/commits/721f11b65029e698acc182c4d20ffc7ec9d5295e

Reply:
thanks!